### PR TITLE
Add support for searching within the same venue in kiosk mode

### PR DESCRIPTION
**Pull request by andreeaceachir142**
# What 

- Add support for searching for locations which are in the same venue in the kiosk mode

# How

- Add `mi-venue` attribute to the `mi-search` component which checks if kiosk we are in kiosk mode
- Add `venue` attribute when getting the filtered locations after clicking on a location 
- Add the `venueId` from the kiosk location to both instances

### CODE REVIEW

File: packages/map-template/src/components/Search/Search.jsx [hunk 1]
```diff
@@ -316,75 +317,49 @@ function Search({ onSetSize, isOpen }) {
...
     }, [useKeyboard]);
 
-
     return (
         <div className="search"
             ref={searchRef}
-            style={calculateContainerStyle()}
-            >
-
-                { /* Search field that allows users to search for locations (MapsIndoors Locations and external) */ }
```
Comment (ammapspeople) [inline packages/map-template/src/components/Search/Search.jsx:326]: Aww, I liked the whitespace that made it easier to read 😢 

Did you change anything else but removing empty lines?

Comment (andreeaceachir142) [reply packages/map-template/src/components/Search/Search.jsx:326]: I knoow... I liked the readability as well, but it's inconsistent with everything else and also a bit hard to maintain when the formatter goes against me 🙃

Comment (andreeaceachir142) [reply packages/map-template/src/components/Search/Search.jsx:326]: Nothing else was changed, except the empty lines.

Comment (ammapspeople) [reply packages/map-template/src/components/Search/Search.jsx:326]: Don't let the editor win against you 😄
It's not inconsistent with everything else: look at LocationDetails.jsx where we have a similar-looking pattern for readability. Also, we need to make the code easier to read.

File: packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx [hunk 2]
```diff
@@ -112,6 +118,7 @@ const SearchField = forwardRef(({ placeholder, mapsindoors, results, clicked, cl
         disabled={disabled}
         mapbox={mapbox}
         google={google}
+        mi-venue={kioskLocation && isKioskContext ? kioskLocation.properties.venueId : undefined}
```
Comment (ammapspeople) [inline packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx:121]: Add a comment explaining that when the Map Template runs in kiosk mode, we restrict searching to the venue only.

File: packages/map-template/src/components/Search/Search.jsx (hunk 1 above)
Comment (ammapspeople) [reply packages/map-template/src/components/Search/Search.jsx:326]: If you just leave a little white space for readability, I'm happy 😄

**Review by ammapspeople — APPROVED** [review]
👏

**Pull request merged**